index.d.ts

```diff
@@ -284,6 +284,11 @@ declare namespace execa {
 			onRejected?: (reason: ExecaError<StdoutErrorType>) => ResultType | PromiseLike<ResultType>
 		): Promise<ExecaReturnValue<StdoutErrorType> | ResultType>;
 
+		/**
+		Same as the original [`child_process.kill()`](https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal) except: if `signal` is `SIGTERM` (the default value) and the child process is not terminated after 5 seconds, force it by sending `SIGKILL`.
+		*/
+		kill(signal?: string | undefined, options?: killOptions | undefined): void;
```

Comment:
You need to add some type tests for this. https://github.com/sindresorhus/execa/blob/master/index.test-d.ts

Reply:
I hope this is sufficient: 353cd0a203a6f2a00ea5482b9f96656da4bb9623